config/crd/bases/ssp.kubevirt.io_ssps.yaml

```diff
@@ -46,31 +50,45 @@ spec:
                   operand
                 properties:
                   url:
-                    description: "URL of a remote Kustomize target from which to generate
-                      and deploy resources. \n The following caveats apply to the
-                      provided URL: \n * Only 'https://' and 'git://' URLs are supported.
-                      \n * The URL must include '?ref=$ref' or '?version=$ref' pinning
-                      it to a specific reference. It is recommended that the reference
-                      be a specific commit or tag to ensure the generated contents
-                      does not change over time. As such it is recommended not to
-                      use branches as the ref for the time being. \n * Only VirtualMachineClusterPreference
-                      and VirtualMachineClusterInstancetype resources generated from
-                      the URL are deployed by the operand. \n See the following Kustomize
-                      documentation for more details: \n remote targets https://github.com/kubernetes-sigs/kustomize/blob/master/examples/remoteBuild.md"
+                    description: |-
+                      URL of a remote Kustomize target from which to generate and deploy resources.
+
```

Comment:
This adds a lot of unneccessary newlines, can it be avoided?

Reply:
I'm not sure how. Maybe there is a config option in the `controller-gen`.

Reply:
No, there isn't any parameter.

Reply:
Maybe a bug in controller-gen?